src/librustc_resolve/lib.rs

```diff
@@ -209,6 +211,14 @@ fn resolve_error<'b, 'a: 'b, 'tcx: 'a>(resolver: &'b Resolver<'a, 'tcx>,
         return;
     }
     match resolution_error {
+        ResolutionError::NameConflictsWithExternCrate(name) => {
+            span_err!(resolver.session,
```

Comment:
This cannot be merged anymore because in master, the type of this match statement has changed. While it still merges cleanly, it will not compile.